src/modules/inspect/cronjob-list/cronjob-list.ts

```diff
@@ -0,0 +1,109 @@
+import {bindable, computedFrom, inject} from 'aurelia-framework';
+
+import {ManagementApiClientService} from '@process-engine/management_api_client';
+import {DataModels} from '@process-engine/management_api_contracts';
+
+import {ISolutionEntry} from '../../../contracts';
+import environment from '../../../environment';
+import {DateService} from '../../../services/date-service/date.service';
+
+@inject('ManagementApiClientService')
+export class CronjobList {
+  @bindable public activeSolutionEntry: ISolutionEntry;
+  public requestSuccessful: boolean = false;
+  public currentPage: number = 1;
+  public pageSize: number = 10;
+
+  private _managementApiService: ManagementApiClientService;
+
+  private _cronjobs: Array<DataModels.Cronjobs.CronjobConfiguration> = [];
+  private _pollingTimeout: NodeJS.Timeout;
+  private _isAttached: boolean;
+
+  constructor(managementApiService: ManagementApiClientService) {
+    this._managementApiService = managementApiService;
+  }
+
+  public async attached(): Promise<void> {
+    this._isAttached = true;
+
+    await this.updateCronjobs();
+    this._startPolling();
+  }
+
+  public detached(): void {
+    this._isAttached = false;
+    this._stopPolling();
+  }
+
+  @computedFrom('_cronjobs.length')
+  public get totalItems(): number {
+    return this._cronjobs.length;
+  }
+
+  @computedFrom('cronjobsToDisplay.length')
+  public get showCronjobList(): boolean {
+    return this.cronjobsToDisplay !== undefined && this.cronjobsToDisplay.length > 0;
+  }
+
+  public get cronjobsToDisplay(): Array<DataModels.Cronjobs.CronjobConfiguration> {
+    const firstCronjobIndex: number = (this.currentPage - 1) * this.pageSize;
+    const lastCronjobIndex: number = (this.pageSize * this.currentPage);
+
+    const cronjobsToDisplay: Array<DataModels.Cronjobs.CronjobConfiguration> =
+      [...this._cronjobs]
```

Comment:
Spreading shouldn't be necessary here.

```suggestion
      this._cronjobs
```

Reply:
Removing this would not break anything, but since `sort` sorts the array in place this would change the value of `_cronjobs` even though this would not be expected.

Imhothe  `cronjobsToDisplay` function should not mutate `_cronjobs`.

Reply:
Fair point.